lib/waterfoul/boot_rom.rb

```diff
@@ -49,10 +49,19 @@ class BootROM
       0x3E, 0x01, 0xE0, 0x50
     ].freeze
 
+    attr_reader :cartridge
```

Comment:
I'm not comfortable about moving this into the `BootRom`. Is there anyway we can move can we keep these separate while keeping the performance gains?